src/content/users/ProfileView.tsx

```diff
@@ -208,7 +214,7 @@ const ProfileView: FC<Props> = ({ _id, viewType }: Props) => {
   // TODO: This is temporary until the API supports sorting natively
```

Comment:
I think this can be removed now since the API should support sorting by studyName?